guides/common/modules/ref_system-requirements.adoc

```diff
@@ -41,6 +41,10 @@ endif::[]
 * A system umask of 0022
 * Full forward and reverse DNS resolution using a fully-qualified domain name
 
+The {Project} and {SmartProxy} server do not support shortnames in the hostnames.
+Also, the custom certificate's Common Name (CN) must have a fully qualified domain name (FQDN) instead of a shortname.
```

Comment:
Please reword to make it active and "Common Name within/of the custom certificate". I am unsure about capitalication.

Reply:
```suggestion
When using custom certificates, the Common Name (CN) of the custom certificate must be a fully qualified domain name (FQDN) instead of a shortname.
```